recipe/meta.yaml

```diff
@@ -28,7 +28,7 @@ test:
     - audioread
 
 about:
-  home: https://github.com/sampsyo/audioread
+  home: https://github.com/beetbox/audioread
```

Comment:
@bmcfee in theory you only need to modify it here. The `feedstock` rendering process will update the README (and other stuff) for you.

Install `conda-smithy` from `conda-forge` with: `conda install -c conda-forge conda-smithy`.

And then do `conda smithy rerender` from the `feedstock` directory. (You still need to commit the changes.)

PS: There is no need to do it now, but you can try that out next time you update the `feedstock`. (And we need to improve our docs to mention `conda-smithy` and the re-rendering process!)